Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml

```diff
@@ -37,6 +37,10 @@ ParserQuery: |
     , ASimAuthenticationPostgreSQL  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationPostgreSQL'  in (DisabledParsers) ))
     , ASimAuthenticationSshd  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationSshd' in (DisabledParsers) ))
     , ASimAuthenticationSu  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationSu' in (DisabledParsers) ))
+    , ASimAuthenticationBarracudaWAF  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationBarracudaWAF' in (DisabledParsers) ))
+    , ASimAuditEventBarracudaWAF  (ASimAuthenticationDisabled or ('ExcludeASimAuditEventBarracudaWAF' in (DisabledParsers) ))
```

Comment:
Only ASimAuthenticationBarracudaWAF should be added in Authentication parser. Others should be placed in respective union parsers.

Reply:
Removed other Barracuda parsers from Authentication and added them in respective unifying parsers in respective PRs.